packages/maps/src/components/basic/GeoDistanceDropdown.js

```diff
@@ -213,6 +213,7 @@ class GeoDistanceDropdown extends Component {
 						this.updateQuery(this.state.currentDistance);
 					}
 					this.locked = false;
+					if (props.onValueChange) props.onValueChange(this.state.currentDistance);
```

Comment:
`onValueChange` should only be called when `this.state.currentDistance` is truthy. So, you should move this statement inside the above `if` block.

Reply:
Also, we need to pass this: as the value in `onValueChange`